modules/dcache/src/main/java/diskCacheV111/poolManager/RequestContainerV5.java

```diff
@@ -1080,15 +1094,7 @@ private boolean answerRequest(int count) {
         //
         // and the heart ...
         //
-        private static final int RT_OK         = 1 ;
-        private static final int RT_FOUND      = 2 ;
-        private static final int RT_NOT_FOUND  = 3 ;
-        private static final int RT_ERROR      = 4 ;
-        private static final int RT_OUT_OF_RESOURCES = 5 ;
-        private static final int RT_COST_EXCEEDED    = 7 ;
-        private static final int RT_NOT_PERMITTED    = 8 ;
-        private static final int RT_S_COST_EXCEEDED  = 9 ;
-        private static final int RT_DELAY  = 10 ;
+
 
         private static final int CONTINUE        = 0 ;
```

Comment:
The way the state machine is implemented here of course could be rewritten, which is not the task at hand, but I am wondering whether a package-visible enum for these two states might not also be desirable.